src/client-commands.ts

```diff
@@ -5,5 +5,6 @@ export const ClientCommands = {
   toggleLogs: "metals-logs-toggle",
   focusDiagnostics: "metals-diagnostics-focus",
   runDoctor: "metals-doctor-run",
-  startDebugSession: "metals-debug-session-start"
+  startDebugSession: "metals-debug-session-start",
+  startRunSession: "metals-run-session-start"
```

Comment:
Is this command only used for testing or also running?

Reply:
for both - a unit (be it class or test) can be either just run or also debugged, hence two commands.

I was thinking about passing additional argument but it would get a bit awkward: there would need to be a separate data structure to hold the `debug: Boolean` field, we would need to extract that on the metals side and separate it from arguments sent to bloop. With this approach we keep it simple: whatever argument this we get is forwarded to bloop as is, without any introspection.

Reply:
Should we rename the "run" part into something else to make it clear it's just "debugging without breakpoints"? For example `metals-debug-no-breakpoints-session-start` 🤔

Reply:
We could, but it would no longer correspond to `run | debug` code lenses or to IntelliJ Run or Debug actions. That was my reasoning - to make it vaguely familiar to the user.

Although, I agree that debugging is also running therefore the name could be better.

Reply:
FWIW, I find "run" and "debug" quite intuitive in their respective meaning of "no breakpoints" / "with breakpoints"